### ISSUE TITLE
the axf should be generated with debug information regardless of the target

### ISSUE THREAD

**Issue by fredo514** [ARM-software/vscode-cmsis-csolution#13]
Type: Feature Request

This maybe be already possible through a way I'm not aware, but it would be nice that since the build command generates a .axf file automatically anyway, it contained the debug information by default without needing to specifically compile in for debug.

**Comment by mcgordonite** [comment]
Hi @fredo514, thanks for the feedback. This issue concerns csolution / cbuild rather than this VS Code extension. Please open an issue on https://github.com/Open-CMSIS-Pack/devtools.

**Comment by fredo514** [comment]
I created an issue to on the other project (https://github.com/Open-CMSIS-Pack/devtools/issues/709) and it turns out there is a csolution flag that is not enabled by default.

I manually added the "debug" option to a "built-types" field under the "solution" in csolution.yaml and with that I was able to launch a debug session without getting an "Unable to find symbols" error or needing to add the "-g" flag to the target compiler options.
```
solution:
  ...
  build-types:
    - type: Release
      debug: "on"
  target-types:
  ...
```
I'm still unable to watch variables and see the current line cursor when I stop on a breakpoint... I'm not sure what I'm missing.
